overlord/snapstate/snapmgr.go

```diff
@@ -1029,6 +1128,23 @@ func (m *SnapManager) startSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	return err
 }
 
+func scheduleNextRefresh(st *state.State) time.Time {
+	randomness := rand.Int63n(int64(refreshRandomness))
+	nextRefreshTime := time.Now().Add(minRefreshInterval).Add(time.Duration(randomness))
+	st.Set("next-auto-refresh-time", nextRefreshTime)
```

Comment:
My impression was that go handles this transparently, i.e. https://play.golang.org/p/slUhw45OYC shows me "+0000 UTC" in the time.Now() string and on my box it shows "+0100 CET". Or am I missing something?